### PR TITLE
add set_state method to multid-widget

### PR DESCRIPTION
**Pull request by tlambert03**
This adds one new public method to the multid widget, which allows you to set the state from an MDASequence object, dict, or yaml file

Also added to `launch-dev` for ease of quickly testing an MDA

### CODE REVIEW

File: micromanager_gui/multid_widget.py [hunk 1]
```diff
@@ -294,6 +289,94 @@ def set_multi_d_acq_dir(self):
         self.dir_lineEdit.setText(self.save_dir)
         self.parent_path = Path(self.save_dir)
 
+    def set_state(self, state: dict | MDASequence | str | Path) -> None:
```
Comment (ianhi) [inline micromanager_gui/multid_widget.py:292]: Similarly would be good to make `get_state` public

Comment (tlambert03) [reply micromanager_gui/multid_widget.py:292]: Should it return a dict or an MDASequence?

Comment (ianhi) [reply micromanager_gui/multid_widget.py:292]: default to sequence with optional arg to make it as a dict?

Comment (tlambert03) [reply micromanager_gui/multid_widget.py:292]: I'll do sequence.  I dislike parameters that change return types.   If someone wants a dict they can always do `get_state().dict()`

Comment (tlambert03) [reply micromanager_gui/multid_widget.py:292]: made it public, added round-trip test

**Pull request merged**